listenbrainz/webserver/views/api.py

```diff
@@ -635,6 +636,33 @@ def get_playlists_collaborated_on_for_user(playlist_user_name):
     return jsonify(serialize_playlists(playlists, playlist_count, count, offset))
 
 
+@api_bp.route("/user/<user_name>/services", methods=['GET', 'OPTIONS'])
+@crossdomain
+@ratelimit()
+def get_service_details(user_name):
```

Comment:
I think this API endpoint should be authenticated and only the current user should be able to access its details.